### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by felixboehm**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -11,42 +11,83 @@ on `localhost:3000`
...
 
-# Download schema
+- file system: `ls -la ~/.storyloc/`
+
+Remove this folder to clean data.
+
+# Download schema for Clients
 
 To download the schema install Apollo CLI.
 On macOS use `brew install apollo-cli`, there are some fixed issued included.
```
Comment (fschade) [inline Readme.md:93]: can we [link](https://github.com/apollographql/apollo-tooling)

File: go.mod [hunk 2]
```diff
@@ -11,37 +11,3 @@ require (
 	github.com/mitchellh/mapstructure v1.4.3
 	github.com/urfave/cli/v2 v2.3.0
 )
-
```
Comment (fschade) [inline go.mod]: its needed and will come back after running `go mod tidy`

**Review by fschade — CHANGES_REQUESTED** [review]
just two small findings, the go mod thing is important

**Comment by fschade** [comment]
@felixboehm any news on that?

File: Readme.md (hunk 1 above)
Comment (felixboehm) [reply Readme.md:93]: yes

**Pull request closed without merging**